sleap/gui/utils.py

```diff
@@ -12,6 +13,7 @@ def is_port_free(port: int, zmq_context: Optional[zmq.Context] = None) -> bool:
     try:
         socket.bind(address)
         socket.unbind(address)
+        time.sleep(0.1)
```

Comment:
Removing `time.wait(0.1)` in this line causes an error while testing `monitor.py` consistently.